src/query/anonymization.c

```diff
@@ -223,32 +223,32 @@ static FuncExpr *make_int8_numeric(List *args, Oid location)
   return cast;
 }
 
-static OpExpr *make_div_operator_common(List *args, Oid location)
+static FuncExpr *make_div_operator_common(List *args, Oid location)
 {
-  OpExpr *division = makeNode(OpExpr);
-  division->opretset = false;
-  division->opcollid = 0;
+  FuncExpr *division = makeNode(FuncExpr);
```

Comment:
`makeFuncExpr` from `makefuncs.h` is more convenient.

Reply:
thanks, I started with that actually, then needed to change for some obsolete reason and finally forgot to change back.